test/import.bats

```diff
@@ -393,3 +393,21 @@ EOF
 
     stacker build
 }
+
+@test "import with dir contents" {
+  mkdir folder1
+  touch folder1/file1
+  cat > stacker.yaml <<EOF
+first:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
```

Comment:
Can you also add a test where the import path is 'stacker://...'?

Reply:
That is going to work yet, will need a part2 of this PR

Reply:
Fixed